requirements/edx/custom.txt

```diff
@@ -25,7 +25,7 @@ git+https://github.com/edx-solutions/xblock-group-project.git@0.1.2#egg=xblock-g
 git+https://github.com/edx-solutions/api-integration.git@v2.5.7#egg=api-integration==2.5.7
 git+https://github.com/edx-solutions/organizations-edx-platform-extensions.git@v1.2.7#egg=organizations-edx-platform-extensions==1.2.7
 git+https://github.com/edx-solutions/gradebook-edx-platform-extensions.git@1.1.13#egg=gradebook-edx-platform-extensions==1.1.13
-git+https://github.com/edx-solutions/projects-edx-platform-extensions.git@v1.1.6#egg=projects-edx-platform-extensions==1.1.6
+git+https://github.com/edx-solutions/projects-edx-platform-extensions.git@agrendalath/MCKIN-8917-delete-group-work#egg=projects-edx-platform-extensions==1.1.8
```

Comment:
Wait, is it 1.1.7 as shown on https://github.com/edx-solutions/projects-edx-platform-extensions/pull/19 ?

Reply:
Yes, I added a migration [here](https://github.com/edx-solutions/projects-edx-platform-extensions/pull/19/commits/41371227b809d763140e504b93d730c3bb267bd7), so I bumped this version from `1.1.7` to `1.1.8` to avoid dependency caching by CI.

Reply:
Got it 👍